sdk/keyvault/Azure.Security.KeyVault.Certificates/perf/Infrastructure/CertificatesScenarioBase.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.Security.KeyVault.Certificates.Perf.Infrastructure
+{
+    public abstract class CertificatesScenarioBase<T> : PerfTest<T> where T : PerfOptions
+    {
+        private readonly Random _rand;
+
+        protected CertificatesScenarioBase(T options) : base(options)
+        {
+            _rand = new Random();
+
+            CertificateClientOptions clientOptions = new()
+            {
+                Transport = PerfTransport.Create(Options),
```

Comment:
Hm, this would cause tests on .NET Framework not to use the default transport and tests on .NET Core not to use default settings.

Reply:
I was merely copying this from other perf tests. What instead do you suggest? /cc @mikeharder

Reply:
Can we run `dotnet dev-certs https --trust` in test environments?

Reply:
Wait, you are hitting the real service so you don't even need to suppress certs.

Reply:
I think we need a bit more logic in PerfTransport to avoid making any changes to settings when they are not requred.

Reply:
`PerfTransport` is still experimental and exists for redirecting to a local mock service.  And it will likely need changes as we implement the new HTTP proxy test server.  Since it's causing confusion I will remove it from the perf framework until it's ready.